base/system/diskpart/diskpart_msg.mc

```diff
@@ -3979,68 +3993,70 @@ Examples:
+                Dimensiunea implicită a unității de alocare pentru un anumit
+                sistem de fișiere este afișată de comanda FILESYSTEMS.
 
-    COMPRESS    NTFS only: Files created on the new volume will be compressed
-                by default.
+                Comprimarea NTFS nu este acceptată pentru dimensiunile
+                unităților de alocare mai mari de 4096.
 
-    OVERRIDE    Forces the file system to dismount first if necessary. All
-                opened handles to the volume would no longer be valid.
+    QUICK       Efectuează o formatare rapidă.
 
-    DUPLICATE   UDF Only: This flag applies to UDF format, version 2.5 or
-                higher.
-                This flag instructs the format operation to duplicate the file
-                system meta-data to a second set of sectors on the disk. The
-                duplicate meta-data is used by applications, for example repair
-                or recovery applications. If the primary meta-data sectors are
-                found to be corrupted, the file system meta-data will be read
-                from the duplicate sectors.
+    COMPRESS    Numai pentru formatul NTFS: Fișierele create pe noul volum
+                vor fi comprimate în mod implicit.
+    OVERRIDE    Forțează mai întâi demontarea sistemului de fișiere, dacă e
+                necesar Toate identificatoarele de gestiune deschise ale
+                volumului nu ar mai fi valabile.
 
-    NOWAIT      Forces the command to return immediately while the format
-                process is still in progress. If NOWAIT is not specified,
-                DiskPart will display format progress in percentage.
+    DUPLICATE   Numai pentru formatul UDF: Această etichetă se aplică
+                formatului UDF, versiunea 2.5 sau mai nouă. Această etichetă
+                intruiește operațiunea de formatare să dubleze metadatele
+                sistemului de fișiere la cel de al doilea set de sectoare
+                de pe disc. Metadatele dublate sunt folosite de aplicații,
+                de exemplu de aplicații de reparare și de recuperare. Dacă
+                sectoarele primare de metadate sunt găsite ca și corupte,
+                metadatele sistemului de fișiere vor fi citite de sectoarele
+                duplicate.
 
-    NOERR       For scripting only. When an error is encountered, DiskPart
-                continues to process commands as if the error did not occur.
-                Without the NOERR parameter, an error causes DiskPart to exit
-                with an error code.
+    NOWAIT      Forțează comanda să se întoarcă imediat atât timp cât procesul
+                de formatare este încă în curs. Dacă NOWAIT nu este
+                specificat, programul DiskPart va afișa progesul formatării
+                în procente.
+                                                
+    NOERR       Doar pentru automatizare. Când apare o eroare, programul
```

Comment:
"For scripting only" gets translated here as "doar pentru automatizare", but in other instances as "doar pentru scriere de fișiere script". Please take care of consistency.